examples/etcd/teardown.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+osc delete all -l template=etcd
```

Comment:
if this is all this does, why not get rid of this script and just document the command in the readme?